src/twisted/web/test/test_http.py

```diff
@@ -2486,7 +2486,8 @@ def test_multipartProcessingFailure(self):
 
     def test_multipartEmptyHeaderProcessingFailure(self):
         """
-        When the multipart does not contain a header is should be skipped
```

Comment:
Is there a section of the RFC that specifies that empty multi-parts are not valid and should be considered bad requests ?

Reply:
RFC 7578 (4.2) "Each part MUST contain a Content-Disposition header field [RFC2183] where the disposition type is "form-data".  The Content-Disposition header field MUST also contain an additional parameter of "name";"

Reply:
Perfect. Thanks. I think that it helps to have a link to the RFC in the docstring for this tests.

But with this, we can consider this a bugfix so we don't have to worry about breaking backward compatibility.